exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/writer/IgnoreCloseOutputStream.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.logging.otlp.internal.writer;
+
+import java.io.FilterOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * An {@link OutputStream} that ignores calls to {@link #close()}. if the underlying stream is
+ * {@link System#out} or {@link System#err}.
+ */
+class IgnoreCloseOutputStream extends FilterOutputStream {
+
+  public IgnoreCloseOutputStream(OutputStream outputStream) {
```

Comment:
#nit: Constructor exceeds the visibility of the class. 
```suggestion
 IgnoreCloseOutputStream(OutputStream outputStream) {
```

Reply:
this class is no longer needed